cli/cli.go

```diff
@@ -308,11 +323,17 @@ func printAPIKeyNotSetMessage() {
 	styleRed := lipgloss.NewStyle().Foreground(lipgloss.Color("9"))
 
 	msg1 := styleRed.Render("OPENAI_API_KEY environment variable not set.")
-	msg2, _ := r.Render(`
+
+	// make it platform agnostic
+	message_string := fmt.Sprintf(`
 	1. Generate your API key at https://platform.openai.com/account/api-keys
 	2. Add your credit card in the API (for the free trial)
 	3. Set your key by running:
-	` + "\n```bash\nexport OPENAI_API_KEY=[your key]\n```\n" + "    4. (Recommended) Add that ^ line to your .zshrc or .bashrc file.\n")
+	%s
+	4. (Recommended) Add that ^ line to your %s file(s).
```

Comment:
nit: want to keep "file" singular instead of "file(s)" (since users will only change one file)